colour/models/rgb/datasets/sony.py

```diff
@@ -208,3 +266,124 @@
 
 S_GAMUT3_CINE_COLOURSPACE : RGB_Colourspace
 """
+
+VENICE_S_GAMUT3_PRIMARIES = np.array([
+    [0.740464264304292, 0.279364374750660],
+    [0.089241145423286, 0.893809528608105],
+    [0.110488236673827, -0.052579333080476],
+])
+"""
+*Venice S-Gamut3* colourspace primaries.
+
+VENICE_S_GAMUT3_PRIMARIES : ndarray, (3, 2)
+"""
+
+VENICE_S_GAMUT3_WHITEPOINT_NAME = S_GAMUT_WHITEPOINT_NAME
+"""
+*Venice S-Gamut3* colourspace whitepoint name.
+
+VENICE_S_GAMUT3_WHITEPOINT_NAME : unicode
+"""
+
+VENICE_S_GAMUT3_WHITEPOINT = S_GAMUT_WHITEPOINT
+"""
+*Venice S-Gamut3* colourspace whitepoint.
+
+VENICE_S_GAMUT3_WHITEPOINT : ndarray
+"""
+
+VENICE_S_GAMUT3_TO_XYZ_MATRIX = normalised_primary_matrix(
+    VENICE_S_GAMUT3_PRIMARIES, VENICE_S_GAMUT3_WHITEPOINT)
+"""
+*Venice S-Gamut3* colourspace to *CIE XYZ* tristimulus values matrix.
+
+VENICE_S_GAMUT3_TO_XYZ_MATRIX : array_like, (3, 3)
+"""
+
+XYZ_TO_VENICE_S_GAMUT3_MATRIX = np.linalg.inv(VENICE_S_GAMUT3_TO_XYZ_MATRIX)
+"""
+*CIE XYZ* tristimulus values to *Venice S-Gamut3* colourspace matrix.
+
+XYZ_TO_VENICE_S_GAMUT3_MATRIX : array_like, (3, 3)
+"""
+
+VENICE_S_GAMUT3_COLOURSPACE = RGB_Colourspace(
+    'Venice S-Gamut3',
+    VENICE_S_GAMUT3_PRIMARIES,
+    VENICE_S_GAMUT3_WHITEPOINT,
+    VENICE_S_GAMUT3_WHITEPOINT_NAME,
+    VENICE_S_GAMUT3_TO_XYZ_MATRIX,
+    XYZ_TO_VENICE_S_GAMUT3_MATRIX,
+    log_encoding_SLog3,
+    log_decoding_SLog3,
+)
+VENICE_S_GAMUT3_COLOURSPACE.__doc__ = """
+*Venice S-Gamut3* colourspace.
+
+References
+----------
+:cite:`SonyElectronicsCorporation2020a`
+
+VENICE_S_GAMUT3_COLOURSPACE : RGB_Colourspace
+"""
+
+VENICE_S_GAMUT3_CINE_PRIMARIES = np.array([
+    [0.775901871567345, 0.274502392854799],
+    [0.188682902773355, 0.828684937020288],
+    [0.101337382499301, -0.089187517306263],
+])
```

Comment:
I had a look to compare the values to the cited reference, and the values there seem to be different. I assume they are in a different format?
If so it would be good to mention the conversion from the source to these values in the doc string.
The same for the `VENICE_S_GAMUT3_CINE_PRIMARIES` ones.

Reply:
Sorry, I just now saw the collab notebook. Maybe that should be referenced here directly.